client/src/app/core/repositories/base-repository.ts

```diff
@@ -336,8 +336,9 @@ export abstract class BaseRepository<V extends BaseViewModel, M extends BaseMode
         });
     }
 
-    protected raiseError = (errorMessage: string | Error) => {
-        this.repositoryServiceCollector.errorService.showError(errorMessage);
+    protected raiseError = (error: Error) => {
+        this.repositoryServiceCollector.errorService.showError(error);
+        throw error;
```

Comment:
Please do not rethrow here. If you rebase, the send*ActionToBackend Methods are adpoted to intercept, rather to catch, the error: https://github.com/OpenSlides/openslides-client/blob/master/client/src/app/core/repositories/base-repository.ts#L343